internal/controller/postgrescluster/pgbackrest.go

```diff
@@ -382,6 +439,24 @@ func unstructuredToRepoResources(kind string, repoResources *RepoResources,
 	uList *unstructured.UnstructuredList) error {
 
 	switch kind {
```

Comment:
There's several changes here because I reordered these cases to match the order of the list where we get these items.